### DISCUSSION TITLE
How does one find out the progress of a download?

### DISCUSSION THREAD

**Discussion by charliep51** [aclap-dev/video-downloadhelper#397] [Help and Q&A]
### Your question

Firefox has a download status feature.  video-downloadhelper should use it.  Right now, it's not obvious how far along the download has proceeded -- or even if the download is working at all.

### Addon & CoApp version and details

Video DownloadHelper
Version 8.2.2.8 (stable)
Browser locale: en-US
Production build
Built on 2024-03-18
Build options: browser=firefox, linuxlic=false, noyt=false, target=mozilla

**Comment by jcfrog** [comment]
hi,
if you click on the blue number you will get a progression bar
<img width="505" alt="Screenshot 2024-04-12 at 19 29 06" src="https://github.com/aclap-dev/video-downloadhelper/assets/368367/d8787851-942e-4ff8-9991-6dcbf380e0a7">
